src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colors.values().length);
+        return Colors.values()[index].toString();
```

Comment:
random should be a class field
Also, a checklist issue:
![image](https://github.com/mate-academy/jv-lottery/assets/49767049/dd6605dc-b6b8-4e12-809d-8f3efea4fb51)